src/main/resources/en-us.tsv

```diff
@@ -1,551 +1,551 @@
-HourlySaveAlert	Medieval Factions is saving. This will happen every hour.
-IntegerSet	Integer set.
-InvitationExpired	Your invitation to %s has expired.
-InvitationSent	Invitation sent.
-KeysLoaded	Keys loaded: %d
-LandAlreadyClaimedBy	This land is already claimed by
-LandAlreadyClaimedByYourFaction	This land is already claimed by your faction.
-LandClaimedBy	This land is claimed by %s.
-LandClaimedUsingAdminBypass	Land unclaimed using admin bypass.
-LandIsNotClaimed	This land isn't claimed.
-LandIsUnclaimed	This land is unclaimed.
-LandNotCurrentlyClaimed	This land is not currently claimed.
-LandUnclaimed	Land unclaimed.
-LawAdded	Law added.
-LawEdited	Law edited.
-LawRemoved	Law removed.
-LawsTitle	== Laws of %s ==
-LeverSuccessfullyLinked	Creating Gate 4/4: Lever successfully linked.
-Liege	Liege: %s
-ListLegend	P: power, M: members, L: land
-LoadingClaimedChunks	Loading claimed chunks...
-LoadingLockedBLocks	Loading locked blocks...
-LoadingPlayerPowerRecords	Loading player power records...
-Locked	Locked.
-LockedBlockSuccessfullyLoaded	Locked block successfully loaded.
-LockedBy	Locked by %s
-LockingCancelled	Locking cancelled.
-LockRemovalError	An error has occurred during lock removal.
-MaterialsMismatch1	Error placing point 1: Materials mismatch.
-MaterialsMismatch2	Error placing point 2: Materials mismatch.
-MedievalFactionsStringRight	Medieval-Factions-%s
-MedievalFactionsTitle	== Medieval Factions %s ==
-MembersOf	Members of %s
-MissingFactionGatesJSONCollection	Could not load gates because the collection 'factionGates' did not exist in the factions JSON file. Are you upgrading from a previous version? Setting default.
-MovementDetectedTeleportCancelled	Movement Detected. Teleport cancelled.
-MustBeAtWarOrFactionMustBeInactive	Must be at war with target faction or target faction must be inactive.
-MustBeInFaction	Sorry, you must be in a faction to use this command.
-MustBeOwner	Sorry, you must be the owner of a faction to use this command.
-MustOwnFactionOrBeOfficer	You have to own a faction or be an officer of a faction to use this command.
-MustTransferOwnership	Sorry. You must transfer ownership or kick everyone in your faction to leave.
-Name	Name: %s
-NameAlreadyTaken	That name is already taken.
-NeutralityDisabled	Neutrality is currently disabled.
-NoBlockDetectedToCheckForGate	No block detected to check for gate.
-NoFactionsDesignatedSingleQuotesRequired	No factions designated. Must be designated inside single quotes.
-NoLongerBypassingProtections	You are no longer bypassing protections provided by Medieval Factions.
-NoLongerInFactionChat	You are no longer in faction chat.
-NotAMemberOfAnyFaction	You are not a member of any faction.
-NotAnAlly	%s isn't an ally of yours.
-NotAnAllyOrVassal	%s isn't an ally or vassal of yours.
-NotAtWarWith	You aren't at war with %s
-NotAVassalOfAFaction	You aren't a vassal of a faction.
+AcceptedChallengeDuelHasBegun                   	You have accepted %s's challenge, the duel has begun.
```

Comment:
Do language key lookups still work with the added spaces here?

Reply:
yes